packages/web/src/components/email/EmailForm.tsx

```diff
@@ -72,7 +72,7 @@ export default function EmailForm() {
         </FormGroup>
       </div>
       <form className="px-2" onSubmit={handleSubmit}>
-        {/* <InputGroup>
+        <InputGroup>
```

Comment:
Disableing Dropdown in From field of Send Email Form.
![image](https://user-images.githubusercontent.com/45717030/158251699-8bc3abeb-be6b-47d4-b452-66546c7ed65f.png)